components/Sections/NewHeader/index.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import Header from './index';
```

Comment:
We should create stories to see active link.
Useful link: [nextjs router storybook](https://storybook.js.org/addons/storybook-addon-next-router)